service/backend/pchain/indexer/parser.go

```diff
@@ -330,20 +313,3 @@ func getProposerFromBytes(bytes []byte) (Proposer, []byte, error) {
 		return Proposer{}, bytes, fmt.Errorf("no handler exists for proposer block type %T", castBlock)
 	}
 }
-
-// initializes tx to have tx identifier generated
-func (p *parser) initializeTx(tx *txs.Tx) error {
```

Comment:
this initialization is not really needed. All the transactions involved come from parsing a block, which guarantees to duly initialize them.
See https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/blocks/block.go#L29:L31